src/main/java/org/generationcp/middleware/dao/GermplasmDAO.java

```diff
@@ -2067,4 +1913,56 @@ private void updateGroupSource(final List<Integer> gids, final Integer newGroupS
 		}
 	}
 
+
+	public List<Variable> getGermplasmAttributeVariables(final List<Integer> gids, final String programUUID) {
```

Comment:
@cuenyad 
I think will be easier to maintain if we use scalars instead of the array index. 
Using sqlQuery.setResultTransformer(Transformers.ALIAS_TO_ENTITY_MAP) will return a Map<String,Object> and the key is the scalar name so it is easier to understand and maintain later

Reply:
@cuenyad 
You missed this comment I guess. 
Please use map here too

Reply:
@clarysabel 
I did, yesterday. I don't now why the change it's not updated.